{"name": "test_MessagesApi02_send_forget_code", "status": "failed", "statusDetails": {"message": "AssertionError: 当天短信验证码超过次数", "trace": "self = <TestCase.Notification.MessagesApi.test_MessagesApi.TestMessagesApi object at 0x00000269C8CF1AF0>\n\n    @allure.story('找回密码发送短信验证码')\n    def test_MessagesApi02_send_forget_code(self):\n        response = zhuorui('消息通知', '找回密码发送短信验证码')\n        assert response.status_code == 200\n        if response.json().get(\"code\") == \"000000\":\n            assert_data(response, '000000', 'ok')\n        # elif response.json().get(\"code\") == \"030002\":\n        #     assert_data(response, '030002', '当天短信验证码超过次数')\n        else:\n>           raise AssertionError(\"当天短信验证码超过次数\")\nE           AssertionError: 当天短信验证码超过次数\n\ntest_MessagesApi.py:38: AssertionError"}, "start": 1597284072139, "stop": 1597284072185, "uuid": "94953671-c426-4382-8292-b10add4ec99e", "historyId": "041f483526c1e2fc233a7956f441b40a", "testCaseId": "96ef388046e4d5362602de33bf767f08", "fullName": "TestCase.Notification.MessagesApi.test_MessagesApi.TestMessagesApi#test_MessagesApi02_send_forget_code", "labels": [{"name": "story", "value": "找回密码发送短信验证码"}, {"name": "feature", "value": "消息通知"}, {"name": "parentSuite", "value": "TestCase.Notification.MessagesApi"}, {"name": "suite", "value": "test_MessagesApi"}, {"name": "subSuite", "value": "TestMessagesApi"}, {"name": "host", "value": "DESKTOP-KBU1G8M"}, {"name": "thread", "value": "92060-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "TestCase.Notification.MessagesApi.test_MessagesApi"}]}